components/table/ProductTableRow.tsx

```diff
@@ -23,7 +27,8 @@ const ProductTableRow = ({ product }: Props) => {
     const bbContainer = bbContentContainer.current;
     if (productCompatibilitiesLength > 0 && bbContainer) {
       const bbImageWidth = bbContainer.clientWidth * 0.75;
-      const numberOfVisibleBBImageinContainer = productCompatibilitiesLength - Math.floor((bbImageWidth || 0) / 28);
+      const numberOfVisibleBBImageinContainer =
+        productCompatibilitiesLength - Math.floor((bbImageWidth || 0) / 28);
```

Comment:
Why 28? I see that this number is also used latter on with setImageSectionWidth. Can we move it to static variable and use it instead of hardcoded number in case of future changes?

Reply:
It's the width of one bb image with padding. I will put it to constant or  dynamically calculate the width of this element